Automattic/wp-calypso
# Revert Fallback behavior for `wpcom-xhr-wrapper`

#### Changes proposed in this Pull Request

* Revert changes to `wpcom-xhr-wrapper` and restore `user/utils` temporarily to address reloading infinitely issue.

#### Testing instructions

1. Boot Calypso Green and make sure you are authenticated
2. Follow description here ( 29993-pb ) to create an invalid token situation
3. Reload Calypso Green
4. Verify user is redirected to login instead of page infinitely reloading.

##### Expected page on invalid token situation
<img width="699" alt="Screen Shot 2021-06-21 at 9 59 37 AM" src="https://user-images.githubusercontent.com/2810519/122800133-6dcc9f80-d277-11eb-923a-2393c732322f.png">




Related to `1164141197617539-as-1200490827699049`
